app/views/privacy-policy.html

```diff
@@ -49,6 +49,12 @@ <h2 class="govuk-heading-m" id="what-personal-data-we-collect-and-why">What pers
 
     <p class="govuk-body govuk-!-margin-bottom-7">If you take part in user research, you may choose to share additional information about you.</p>
 
+    <p class="govuk-body govuk-!-margin-bottom-7">We use Google Analytics software to collect information about how you use Register trainee teachers. This includes IP addresses. It helps us to improve the service.</p>
+
+    <p class="govuk-body govuk-!-margin-bottom-7">We do not store your personal information through Google Analytics. The data is anonymised before being used for analytics processing.</p>
+
+    <p class="govuk-body govuk-!-margin-bottom-7"><a href="https://register-prototype.herokuapp.com/cookies" class="govuk-link">Find out more about our cookie policy</a></p>
```

Comment:
```suggestion
    <p class="govuk-body govuk-!-margin-bottom-7"><a href="/cookies" class="govuk-link">Find out more about our cookie policy</a></p>
```

We generally want to avoid using the 'public' url as this wouldn't work if the app were deployed anywhere else - like in review apps.